### PR TITLE
emea nov 2021

### PR DESCRIPTION
**Pull request by jennyf19**

### CODE REVIEW

File: EMEA-2021-AppIntegrationMSAL.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
+# EMEA | App Integration with AzureAD B2C and MSAL | Novemeber 2021
+
+## B2C & App Integration with MSAL
+
+**In this article**
+- [Prerequisites](#prerequisites)
+- [Lab 01: MSAL JS Angular Single-Page Application](#lab-01-msal-js-angular-single-page-application)
```
Comment (jmprieur) [inline EMEA-2021-AppIntegrationMSAL.md]: we need to check the instructions as it now uses MSAL 2.x

File: EMEA-2021-AppIntegrationMSAL.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+1) Click "Logout" to logout from the application.
+
+
+## Lab 02: Create your own web app
+
+### Install the Microsoft.Identity.Web templates in .NET Core
+
+In this lab, we'll need to use the Microsoft.Identity.Web project templates (which are also available in .NET 5.0 from Preview 7). To install them in your system:
+
+1. Download the [Microsoft.Identity.Web.ProjectTemplates-1.1.0](https://www.nuget.org/api/v2/package/Microsoft.Identity.Web.ProjectTemplates/1.1.0) NuGet package from NuGet.org, and save it locally
```
Comment (jmprieur) [inline EMEA-2021-AppIntegrationMSAL.md]: We need to update the version to the latest

File: EMEA-2021-AppIntegrationMSAL.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+1. In a developer command prompt, type
+   ```dotnetcli
+   dotnet new -i Microsoft.Identity.Web.ProjectTemplates.1.1.0.nupkg
```
Comment (jmprieur) [inline EMEA-2021-AppIntegrationMSAL.md]: same

**Review by jmprieur — COMMENTED** [review]
LGTM
A few things to change

**Comment by jennyf19** [comment]
@jmprieur please re-review

File: EMEA-2021-AppIntegrationMSAL.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+ - [B2C Test Tenant](https://docs.microsoft.com/en-us/azure/active-directory-b2c/tutorial-create-tenant). You can configure your own B2C tenant and use that configuration.
+ - We'll be using Visual Studio 2022.
+
+## Lab 01: Create your own web app
+
+### Install the Microsoft.Identity.Web templates in .NET Core
+
+In this lab, we'll need to use the Microsoft.Identity.Web project templates (which are also available in .NET 5.0 from Preview 7). To install them in your system:
+
+1. Download the [Microsoft.Identity.Web.ProjectTemplates-1.20.0](https://www.nuget.org/api/v2/package/Microsoft.Identity.Web.ProjectTemplates/1.20.0) NuGet package from NuGet.org, and save it locally
```
Comment (jmprieur) [inline EMEA-2021-AppIntegrationMSAL.md]: Not even needed. The .NET 6.0 or .NET 5.0 templates should work?

File: EMEA-2021-AppIntegrationMSAL.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+1. Create a folder, and cd into it.
+   ```bash
+   mkdir webapp
+   cd webapp
+   ```
+
+1. Create the B2C web app. Use the clientId, instance and domain for the app you created in the Azure AD B2C portal.
+   ```dotnetcli
+   dotnet new webapp2 --auth IndividualB2C ^
```
Comment (jmprieur) [inline EMEA-2021-AppIntegrationMSAL.md]: and in that case dotnet new webapp (not 2)

**Review by jmprieur — APPROVED** [review]

**Pull request merged**